mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -97,6 +97,22 @@ class LinearRegression(override val uid: String)
   setDefault(standardization -> true)
 
   /**
+   * Set the robust Option to determine whether to use robust Huber Cost Function
```

Comment:
nit: This is not really an "Option", can we just make this say " Set whether to use robust Huber Cost Function"